### DISCUSSION TITLE
如何解决公网下载本地盘的文件

### DISCUSSION THREAD

**Discussion by Liqiuking** [AlistGo/alist#2228] [General]
试了下本地搭建，无论是本地盘还是云盘都能下载文件。慢慢开始第二步挂载在自己网盘在互联网上，经多次测试域名是可以访问的，但是无法下载挂载在本地盘的文件，处了本地盘我还挂载了阿里云盘，阿里云盘可以下载。
经测试，本地盘上在公网上(域名访问)无法下载，但是可以访问；在内网(localhost)上可以下载，可以访问。
阿里盘在公网上(域名访问)可以下载，可以访问；在内网(localhost)上可以下载，可以访问。
应该不是权限问题，admin账户一样无法下载本地盘的文件
通过nginx代理，https://suidao.io/#/这个软件打隧道，https://dash.cloudflare.com/  发布域名
该网页无法正常运作
该网页无法正常运作  域名.tk 目前无法处理此请求。
HTTP ERROR 502

**Comment by rakaki** [comment]
我没这个问题。用的一键脚本装的alist，ipv6 ddns

**Comment by Vincistudio** [comment]
我也没这个问题 排除alist的问题吧

**Comment by Ryan-xh** [comment]
我使用ddnsto服务映射的本地端口，同样无法下载可以访问。但是使用ipv6的ddns到域名，就是可以正常下载。本地也能正常下载。所以排除了alist的问题。问题应该是出在内网穿透上

**Comment by Fan-0104** [comment]
alist 首页 站点url 填入你域名和端口就好
